.github/workflows/ci.yml

```diff
@@ -8,10 +8,10 @@ on:
 
 env:
   DOCKER_TAG: buildenv
-  DOCKER_FILE_PATH: ./docker-build-env/Dockerfile
+  DOCKER_FILE_PATH: ./dockerfiles/base
 
   DOCKER_WITH_HSM_TAG: buildenv_with_hsm
-  DOCKER_WITH_HSM_FILE_PATH: ./docker-build-env/Dockerfile-With-HSM
+  DOCKER_WITH_HSM_FILE_PATH: ./dockerfiles/hsm-support
```

Comment:
minor: The variable name is now slightly lying to you as it isn't the file path anymore but rather like some directory.

Reply:
Agreed, I used `folder` instead. Context might be too abstract for a casual reader.